src/app/rpmostree-dbus-helpers.c

```diff
@@ -833,9 +845,14 @@ get_modifiers_variant (const char   *set_refspec,
     g_variant_dict_insert (&dict, "set-refspec", "s", set_refspec);
   if (set_revision)
     g_variant_dict_insert (&dict, "set-revision", "s", set_revision);
-  if (uninstall_pkgs != NULL && g_strv_length ((char**)uninstall_pkgs) > 0)
-    g_variant_dict_insert (&dict, "uninstall-packages", "^as",
-                           (char**)uninstall_pkgs);
+
+  /* NB: when we implement this, we'll have to pass it through sort_pkgs_strv to
+   * split out local pkgs */
+  vardict_insert_strv (&dict, "override-replace-packages", override_replace_pkgs);
+
+  vardict_insert_strv (&dict, "uninstall-packages", uninstall_pkgs);
```

Comment:
Out of curiosity, why is `uninstall` here in the middle of the `override-` section?